src/main/python/ttconv/imsc/attributes.py

```diff
@@ -133,7 +133,7 @@ def extract(ttml_element) -> model.CellResolutionType:
 
   @staticmethod
   def set(ttml_element, res):
```

Comment:
Where does the writing of px happen now?

Reply:
Cell resolution is in number of row and columns, so no units.

See https://www.w3.org/TR/ttml2/#parameter-attribute-cellResolution